core/templates/domain/question/question-update.service.spec.ts

```diff
@@ -16,86 +16,31 @@
  * @fileoverview Unit tests for question update service.
  */
 
-// TODO(#7222): Remove the following block of unnnecessary imports once
-// question-update.service.ts is upgraded to Angular 8.
-import { AnswerGroupObjectFactory } from
-  'domain/exploration/AnswerGroupObjectFactory';
-import { HintObjectFactory } from 'domain/exploration/HintObjectFactory';
-import { OutcomeObjectFactory } from
-  'domain/exploration/OutcomeObjectFactory';
-import { ParamChangeObjectFactory } from
-  'domain/exploration/ParamChangeObjectFactory';
-import { ParamChangesObjectFactory } from
-  'domain/exploration/ParamChangesObjectFactory';
-import { RuleObjectFactory } from 'domain/exploration/RuleObjectFactory';
-import { UnitsObjectFactory } from 'domain/objects/UnitsObjectFactory';
-import { WrittenTranslationObjectFactory } from
-  'domain/exploration/WrittenTranslationObjectFactory';
-import { WrittenTranslationsObjectFactory } from
-  'domain/exploration/WrittenTranslationsObjectFactory';
-import { UpgradedServices } from 'services/UpgradedServices';
-// ^^^ This block is to be removed.
-// TODO(#7222): Remove usage of importAllAngularServices once upgraded to
-// Angular 8.
 import { importAllAngularServices } from 'tests/unit-test-utils.ajs';
 import { SubtitledHtml } from 'domain/exploration/subtitled-html.model';
-
-require('App.ts');
-require('domain/editor/undo_redo/question-undo-redo.service.ts');
-require('domain/question/QuestionObjectFactory.ts');
-require('domain/question/question-update.service.ts');
-require('domain/state/StateObjectFactory.ts');
-require(
-  'components/question-directives/question-editor/' +
-  'question-editor.component.ts');
+import { TestBed } from '@angular/core/testing';
+import {QuestionUpdateService} from 'domain/question/question-update.service';
```

Comment:
Please keep the formatting consistent. Spaces after { and before }... Here and elsewhere.

Reply:
done